temporalio/test/test.rb

```diff
@@ -72,6 +74,10 @@ def skip_if_fibers_not_supported!
     skip('Fibers not supported in this Ruby version')
   end
 
+  def skip_if_not_x86!
+    skip('Test only supported on x86') unless RbConfig::CONFIG['host_cpu'] == 'x86_64'
```

Comment:
nit: I'd suggest making this predicate more targeted, i.e. `skip_if_test_server_not_supported` and allow arm64 on macos.

Most devs in the team are working on Apple M* cpus, and as it is now, they wouldn't be running those tests localy even though they technically could. I know there's very few tests that depends on that condition, but still…

Reply:
Makes sense, will change